tensorflow/lite/g3doc/guide/build_cmake.md

```diff
@@ -57,18 +57,65 @@ provide `-DCMAKE_BUILD_TYPE=Debug` option.
 cmake ../tensorflow_src/tensorflow/lite -DCMAKE_BUILD_TYPE=Debug
 ```
 
-#### Cross-compilation for Android
+#### Build with kernel unit tests
 
-You can use CMake to build Android binaries. You need to install
-[Android NDK](https://developer.android.com/ndk) and provide the NDK path with
-`-DDCMAKE_TOOLCHAIN_FILE` flag. You also need to set target ABI with
-`-DANDROID_ABI` flag.
+In order to be able to run kernel tests, you need to provide
+'-DTFLITE_KERNEL_TEST=on' flag. Unit test cross-compilation specifics
+can be found in the next subsection.
+
+```sh
+cmake ../tensorflow_src/tensorflow/lite -DTFLITE_KERNEL_TEST=on
+```
+#### Cross-compilation
+
+You can use CMake to build binaries for ARM64 or Android target architectures.
+
+In order to cross-compile the TF Lite, you namely need to provide the path
+to the SDK (e.g. ARM64 SDK or NDK in Android's case) with `-DCMAKE_TOOLCHAIN_FILE` flag.
+
+```sh
+cmake -DCMAKE_TOOLCHAIN_FILE=<CMakeToolchainFileLoc> ../tensorflow/lite/ 
+```
+
+##### Specifics of Android cross-compilation
+
+For Android cross-compilation, you need to install [Android NDK]
+(https://developer.android.com/ndk) and provide the NDK path with `-DCMAKE_TOOLCHAIN_FILE`
+flag mentioned above. You also need to set target ABI with`-DANDROID_ABI` flag.
 
 ```sh
 cmake -DCMAKE_TOOLCHAIN_FILE=<NDK path>/build/cmake/android.toolchain.cmake \
   -DANDROID_ABI=arm64-v8a ../tensorflow_src/tensorflow/lite
 ```
 
+##### Specifics of kernel (unit) tests cross-compilation
+
+Cross-compilation of the unit tests requires flatc compiler for the host architecture.
+For this purpose, there is a CMakeLists located in _tensorflow/lite/tools/cmake/native_tools/flatbuffers_
+to build the flatc compiler with CMake in advance using the host toolchain:
+
+```sh
+cmake ../tensorflow_src/tensorflow/lite/tools/cmake/native_tools/flatbuffers
+cmake --build .
+```
+
+It is also possible to install the _flatc_ into a custom installation location
+(e.g. containing other natively-built tools):
+
+```sh
+cmake -DCMAKE_INSTALL_PREFIX=<native_tools_dir> ../tensorflow_src/tensorflow/lite/tools/cmake/native_tools/flatbuffers
+cmake --build .
```

Comment:
don't we need to run install command?

Reply:
No, running **cmake --build .** installs the flatc automatically.